matchmaker/views/external_api.py

```diff
@@ -160,9 +161,9 @@ def _generate_notification_for_incoming_match(results, incoming_query, incoming_
             if email.strip() != MME_DEFAULT_CONTACT_EMAIL]
         all_emails.update(emails)
         match_results.append((result_text, emails))
-    match_results = sorted(match_results, key=lambda (result_text, _): result_text)
+    match_results = sorted(match_results, key=lambda result_text__: result_text__[0])
```

Comment:
Can you not destructure tuple args in python 3? That seems like a weird thing to get rid if. If we do need to change this, the variable name `result_text__` is a bit confusing. Maybe just use `result_tuple` or something like that?

Reply:
Python 3 doesn't support interpret the members of the tuple args. The only way is to use the index to access the tuple members. I can change the name to `result_tuple`.